style.css

```diff
@@ -638,3 +321,163 @@ textarea {
   line-height: 28px;
   padding-left: 10%;
 }
+
+.hide-in-mobile {
+  display: none;
+}
+
+@media only screen and (min-width: 990px) {
+  .hide-in-desktop {
+    display: none;
+  }
+
+  .show {
+    display: block;
+  }
+
+  .navbarlist {
+    display: flex;
+  }
+
+  ul,
+  li a {
+    text-decoration: none;
+    color: #f9f9f9;
+    margin-right: 20px;
+  }
+
+  .logo > a {
+    margin-left: 15px;
+  }
+
+  .first-section {
+    background-image: url("IMG1/desktop-background.png");
+    background-repeat: no-repeat;
+    background-size: cover;
+  }
+
+  .header-container h2 {
+    font-size: 90px;
+    line-height: 105px;
+    margin: 10%;
+    width: 100%;
+    height: 96px;
+  }
+
+  .header-container h3 {
+    font-size: 40px;
+    line-height: 47px;
+    width: 1156px;
+    height: 72px;
+    margin: 24px 0;
+    text-align: center;
+  }
+
+  .intro-container {
+    display: grid;
+    grid-template-columns: 1fr;
+    margin: 0 33%;
+  }
+
+  .top-divider {
+    width: 120%;
+  }
```

Comment:
- [ ] Great job so far. However, this is a bit different from the design the language frameworks and skills should not get out of the top and bottom lines Please fix that.

### Your Implementation
![image](https://user-images.githubusercontent.com/59416136/155730438-04a0e362-0289-4dfa-9bab-ec9a0ce33f71.png)


## Figma's Design
![image](https://user-images.githubusercontent.com/59416136/155730531-4fd747b9-e52d-4887-82e0-47ad9ac9cb8e.png)